clustered/server/src/main/java/org/ehcache/clustered/server/store/ClusterTierActiveEntity.java

```diff
@@ -653,14 +607,7 @@ public void destroy() {
   }
 
   Set<ClientDescriptor> getConnectedClients() {
-    return unmodifiableSet(connectedClients.keySet());
```

Comment:
If we need to know the set of connected clients for non-management reasons then we shouldn't be relying on management to track that set.  I think we're going to need to maintain a set of connected clients here still (unless Voltron can provide a `messageAllClients(...)` API.

Reply:
So you abandon your push up in management idea?

Reply:
In `ClusterTierManagerActiveEntity` it is only used to dump the state. In `ClusterTierActiveEntity` it is used to dump the state.

Reply:
We can push the map up... but it looks like we'll need to keep a set in the active entity to handle the invalidation logic.  That set should be temporary since eventually it will get replaced with a specialized statistical set so that we can do smarter/lazier invalidations.

Reply:
Ok. I've moved it back to `ClusterTierActiveEntity`. And removed it from `ClusterTierManagerActiveEntity` since it's not needed. It should be alright now.